deployment_specific/pb/ds_locales/en.js

```diff
@@ -171,63 +171,58 @@ const en = {
       'title': 'Watch Instructional Video'
     },
     'whatsapp': {
-      'title': 'Report via WhatsApp',
+      'title': 'Complete Training Exercise via WhatsApp',
       'steps': [
-        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or <a target="_blank" href="https://bit.ly/BencanaBotWA">https://bit.ly/BencanaBotWA</a>',
+        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or https://bit.ly/BencanaBotWA',
         'DisasterBot will send you a one-time link to submit a disaster report.',
-        'Open the link and follow the instructions to input hazard location, upload photo and description.',
-        'DisasterBot will send you a reply with a link to your report.',
-        'Check PetaBencana.id on your browser for up-to-date disaster information.'
+        'Open the link, select “complete training exercise”, and follow the instructions to input hazard location, upload photo and description.',
+        'DisasterBot will send you a reply with a link to your simulation report.'
       ]
     },
     'twitter': {
-      'title': 'Report Via Twitter',
+      'title': 'Complete Training Exercise via X (formerly Twitter)',
       'steps': [
-        'Tweet <a href="https://twitter.com/intent/tweet?text=@petabencana">@petabencana</a> with one of the following hashtags:<ul><li>#flood</li><li>#earthquake</li><li>#forestfire</li><li>#haze</li><li>#volcano</li></ul>',
+        'Tweet @petabencana with one of the following hashtags:<ul><li>#flood</li><li>#earthquake</li><li>#forestfire</li><li>#haze</li><li>#volcano</li></ul>',
```

Comment:
Make sure to have petabencana tag to be in ahref look at the change in red it should be like that